pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -474,6 +481,9 @@ func (r *RepoSyncReconciler) Register(mgr controllerruntime.Manager, watchFleetM
 		// Watch RoleBindings in all namespaces, because RoleBindings are created
 		// in the namespace of the RepoSync. Only maps to existing RepoSyncs.
 		Watches(&source.Kind{Type: &rbacv1.RoleBinding{}},
+			handler.EnqueueRequestsFromMapFunc(r.mapObjectToRepoSync),
+			builder.WithPredicates(predicate.ResourceVersionChangedPredicate{})).
+		Watches(&source.Kind{Type: &admissionv1.ValidatingWebhookConfiguration{}},
 			handler.EnqueueRequestsFromMapFunc(r.mapObjectToRepoSync),
```

Comment:
You can't reuse the `mapObjectToRepoSync` function here.
Basically, the function should map the `admission-webhook.configsync.gke.io` ValidatingWebhookConfiguration object to all RepoSync objects.

You'll need a separate mapper.

Reply:
Done!